README.md

```diff
@@ -1,62 +1,62 @@
 
 Python scripting
 ================
 
-Invoking cddl_gen.py from the command line
-------------------------------------------
+Invoking zcbor.py from the command line
+---------------------------------------
 
-The cddl_gen.py script can directly read CBOR, YAML, or JSON data and validate it against a CDDL description.
+The zcbor.py script can directly read CBOR, YAML, or JSON data and validate it against a CDDL description.
 It can also freely convert the data between CBOR/YAML/JSON.
 It can also output the data to a C file formatted as a byte array.
 
 The following is a generalized example for converting (and validating) data from the command line.
 The script infers the data format from the file extension, but the format can also be specified explicitly.
-See `cddl-gen convert --help` for more information.
+See `ZCBOR convert --help` for more information.
 
 ```sh
-python3 <cddl-gen base>/cddl_gen/cddl_gen.py -c <CDDL description file> convert -t <which CDDL type to expect> -i <input data file> -o <output data file>
+python3 <ZCBOR base>/zcbor/zcbor.py -c <CDDL description file> convert -t <which CDDL type to expect> -i <input data file> -o <output data file>
 ```
 
 Or invoke its command line executable (if installed via `pip`):
 
 ```sh
-cddl-gen -c <CDDL description file> convert -t <which CDDL type to expect> -i <input data file> -o <output data file>
+zcbor -c <CDDL description file> convert -t <which CDDL type to expect> -i <input data file> -o <output data file>
 ```
 
-Note that since CBOR supports more data types than YAML and JSON, cddl-gen uses an idiomatic format when converting to/from YAML/JSON.
+Note that since CBOR supports more data types than YAML and JSON, ZCBOR uses an idiomatic format when converting to/from YAML/JSON.
 This is relevant when handling YAML/JSON conversions of data that uses the unsupported features.
 The following data types are supported by CBOR, but not by YAML (and JSON which is a subset of YAML):
 
  1. bytestrings: YAML supports only text strings. In YAML, bytestrings ('<bytestring>') are represented as {"bstr": "<hex-formatted bytestring>"}, or as {"bstr": <any type>} if the CBOR bytestring contains CBOR-formatted data, in which the data is decoded into <any type>.
  2. map keys other than text string: In YAML, such key value pairs are represented as {"keyval<unique int>": {"key": <key, not text>, "val": <value>}}
  3. tags: In cbor2, tags are represented by a special type, cbor2.CBORTag. In YAML, these are represented as {"tag": <tag number>, "val": <tagged data>}.
 
-Importing cddl_gen in a Python script
--------------------------------------
+Importing zcbor in a Python script
+----------------------------------
 
-Importing cddl_gen gives access to the DataTranslator class which is used to implement the command line conversion features.
+Importing zcbor gives access to the DataTranslator class which is used to implement the command line conversion features.
```

Comment:
Here we reference the project (or the python package) in lower-case. One could argue that it is right here since its the python package that is being referenced (which would be lower case). But this just shows that it is hard to be consistent if the name of the project is different from the executable and libraries etc.